rs/nns/governance/src/storage/neurons.rs

```diff
@@ -823,27 +823,30 @@ fn update_repeated_field<Element, Memory>(
 // TODO(NNS1-2513): To avoid the caller passing an incorrect range (e.g. too
 // small, or to big), derive range from NeuronId.
 fn update_range<Key, Value, Memory>(
```

Comment:
note for reviewer: this is the main optimization to how we handle storing things